content/applications/general/in_app_purchase.rst

```diff
@@ -2,58 +2,63 @@
 In-App Purchase (IAP)
 =====================
 
-In-App Purchases (IAP) gives access to additional services through Odoo. For instance, it allows you
-to send SMS Text Messages or to send Invoices by post directly from your database.
+In-App Purchases (IAP) grant access to additional services through Odoo. For instance, it allows the
+sending of SMS text messages or invoices by post directly from a database.
 
-.. _iap/buying_credits:
+.. _Buy credits:
 
-Buying Credits
+Buying credits
 ==============
 
-Each IAP Service relies on prepaid credits to work and has its own pricing. To consult your current
-balance or to recharge your account, go to :menuselection:`Settings --> Odoo IAP --> View my
-Services`.
+Each IAP service relies on prepaid credits to work and has its own pricing. To check an account’s
+current balance or to buy more credits (:guilabel:`Recharge`), go to the
+:menuselection:`Settings app --> Search "IAP" --> View My Services`.
 
-.. image:: in_app_purchase/image1.png
+.. image:: in_app_purchase/search-for-iap-in-settings.png
    :align: center
+   :alt: General Settings and the In-App Purchases heading showing the View My Services button.
 
 .. tip::
-   If you are on Odoo Online and have the Enterprise version, you benefit from free credits to test our
-   IAP features.
+   Users who have the Enterprise version of Odoo Online get free credits to test IAP features.
 
 IAP accounts
 ============
 
-Credits to use IAP services are stored on IAP accounts, which are specific to each service.
-By default, IAP accounts are common to all companies, but can be restricted to specific
-ones. Activate the :ref:`developer mode <developer-mode>`, then go to :menuselection:`Technical
-Settings --> IAP Account`.
+Credits for IAP services are stored on IAP accounts, which are specific to each service. By default,
+IAP accounts are common to all companies, but can be restricted to specific ones. To make changes to
+IAP accounts, activate Odoo’s :ref:`developer mode <developer-mode>`. Return to the main apps page
+and type *IAP*. In the resulting dropdown menu select
+:guilabel:`Settings / Technical / IAP / IAP Accounts`.
 
 .. image:: in_app_purchase/image2.png
    :align: center
+   :alt: The IAP Account menu showing IAP services, companies, and account tokens.
 
 .. tip::
-   An IAP account can be disabled by appending `+disabled` to its token.
-   Reverting this change will re-enable the account.
+   To disable an IAP account, append `+disabled` to its token. To re-enable the account, remove
+   `+disabled` from the token.
```

Comment:
where? technical menu IAP-> IAP accounts (in dev mode)

Reply:
Expanded this tip to be part of the overall IAP accounts section, added content for making changes and fuller steps for disabling an account. Also added a new image showing "+disabled" at the end of an account token.